src/main/java/roomescape/controller/api/ReservationRestController.java

```diff
@@ -0,0 +1,62 @@
+package roomescape.controller.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import roomescape.dto.CreateReservationRequest;
+import roomescape.dto.GetReservationResponse;
+import roomescape.model.Reservation;
+import roomescape.repository.MemoryReservationRepository;
+import roomescape.repository.ReservationRepository;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationRestController {
+
+    private final ReservationRepository reservationRepository = new MemoryReservationRepository();
+
+    @GetMapping("")
```

Comment:
```suggestion
    @GetMapping
```

생략도 가능합니다.

[숙제] 금번 미션은 여유가 있을테니, 이번 미션에서 처음 사용한 애너테이션, 클래스 등에 대해 어떻게 사용하는 방법들이 있는지, 대체할 수 있는 방법들이 있는지, 적합한 사용방법은 어떤 것인지 조금 더 깊이있게 알아보고, 정리해보세요.

Reply:
감사합니다! 해당 숙제에 대해서는 이번 미션1이 끝날 때까지 열심히 작성해 보겠습니다!